### DISCUSSION TITLE
Coinbase Wallet Connect Bugs

### DISCUSSION THREAD

**Discussion by wpcorbett79** [WalletConnect/walletconnect-monorepo#4190] [Developer Support]
Hi, with the current version of Web3 Modal, the Coinbase Wallet mobile connection is no longer working. The previous version opened my dapp within the Coinbase browser which allowed my users to mint NFTs with CB mobile. Is there a solution so that Coinbase wallet on Mobile can be used?

Also, the desktop extension QR code for Coinbase opens to MetaMask (see attached image)
![IMG_6384](https://github.com/WalletConnect/walletconnect-monorepo/assets/147431153/39ced44b-f6aa-45ab-a7a8-26cc317b5d6c)

**Comment by glitch-txs** [comment]
Looks like a wrong setup, you can check labs in the Web3Modal repo how it should be implemented

**Comment by 93lucasp** [comment]
> Hi, with the current version of Web3 Modal, the Coinbase Wallet mobile connection is no longer working. The previous version opened my dapp within the Coinbase browser which allowed my users to mint NFTs with CB mobile. Is there a solution so that Coinbase wallet on Mobile can be used?
> 
> Also, the desktop extension QR code for Coinbase opens to MetaMask (see attached image) ![IMG_6384](https://private-user-images.githubusercontent.com/147431153/301026876-39ced44b-f6aa-45ab-a7a8-26cc317b5d6c.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0NDQzNDgsIm5iZiI6MTcwODQ0NDA0OCwicGF0aCI6Ii8xNDc0MzExNTMvMzAxMDI2ODc2LTM5Y2VkNDRiLWY2YWEtNDVhYi1hN2E4LTI2Y2MzMTdiNWQ2Yy5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMjBUMTU0NzI4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDQyYmUwNzdlYjI0NmVkOThkMzdjMGRkNGZmNzI1YTJlZjE4NzQ2NWUxNDE5YWZkMmM1ODZhNTU0ZjA1ODg4MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Hyl--L7VHBP07EcnxJlaJxSFUkc_r8dyNe_u6ZqR5lw)

same issue here

**Comment by wpcorbett79** [comment]
The connect button on the Web3 Modal website works perfectly. So maybe Wallet Connect fixed the issue with version 3.5.6?

Best regards,

William Corbett
Founder & CEO
Ethereum Address Service (EAS)
www.easnft.com<https://www.easnft.com/>

Book a Meeting with Me<https://meetings.hubspot.com/william-corbett>


On Feb 20, 2024, at 10:49 AM, Luca Spezzano ***@***.***> wrote:

﻿

Hi, with the current version of Web3 Modal, the Coinbase Wallet mobile connection is no longer working. The previous version opened my dapp within the Coinbase browser which allowed my users to mint NFTs with CB mobile. Is there a solution so that Coinbase wallet on Mobile can be used?

Also, the desktop extension QR code for Coinbase opens to MetaMask (see attached image) [IMG_6384] <https://private-user-images.githubusercontent.com/147431153/301026876-39ced44b-f6aa-45ab-a7a8-26cc317b5d6c.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0NDQzNDgsIm5iZiI6MTcwODQ0NDA0OCwicGF0aCI6Ii8xNDc0MzExNTMvMzAxMDI2ODc2LTM5Y2VkNDRiLWY2YWEtNDVhYi1hN2E4LTI2Y2MzMTdiNWQ2Yy5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMjBUMTU0NzI4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDQyYmUwNzdlYjI0NmVkOThkMzdjMGRkNGZmNzI1YTJlZjE4NzQ2NWUxNDE5YWZkMmM1ODZhNTU0ZjA1ODg4MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Hyl--L7VHBP07EcnxJlaJxSFUkc_r8dyNe_u6ZqR5lw>

same issue here
